src/cmd/openmldb.cc

```diff
@@ -368,9 +368,6 @@ ::openmldb::base::Status PutSchemaData(const ::openmldb::nameserver::TableInfo&
         return ::openmldb::base::Status(-1, "Encode data error");
     }
     std::vector<uint64_t> ts_dimensions;
-    if (codec.EncodeTsDimension(input_value, &ts_dimensions) < 0) {
-        return ::openmldb::base::Status(-1, "Encode ts dimension error");
-    }
```

Comment:
* delete `ts_dimensions` parameter in  `PutData`
* delete put method with ts_dimensions in client/tablet_client.h